### ISSUE TITLE
BUG - 3 - All - Accessibility: Screen reader adjustments for Delayed Refill message

### ISSUE THREAD

**Issue by SKMOstudio** [department-of-veterans-affairs/va-mobile-app#13159]
## What happened?

For the Delayed Refill message on the Prescriptions page, there are a few inconsistencies that need to be adjusted fro Android and iOS for screen reader users:

iOS - The screen reader is focusing on the bullet point symbol for each listed item in the Delayed Prescription message.

Android - When a screen reader user taps on a link to a prescription, the user is directed to the prescription detail page, but the screen reader focuses on the delayed refill message as the first item on the page. For consistency with how the user navigates between other pages, the first focused item should be the 'Back' button at the top of the page, regardless of the urgency of the message.

## Specs:

- Device: Android, iOS
- OS: Latest
- User Account: ELENA.GPRSYSTESTSEVEN@id.me
- Accessibility State: Screen reader ON, default settings.

## Steps to Reproduce

iOS:
1. Log into the app as the test user.
2. Turn the screen reader ON.
3. From the Home page, navigate to the Health > Prescriptions page.
4. Navigate to the Delayed Refill message box. ("Some refills are taking longer than expected...")
5. Within the message box, swipe-navigate to focus the screen reader on the bullet list items of delayed medications.
6. Observe: The screen reader is focusing and reading out "bullet" for each bullet point item in the list.

Android:
1. Log into the app as the test user.
2. Turn the screen reader ON.
3. From the Home page, navigate to the Health > Prescriptions page.
4. Focus the screen reader on the link to a prescription detail, then tap it to go to that page.
5. Observe: The user is directed to the prescription detail page, but the screen reader focus jumps to the Delayed Refill message box, instead of the 'Back' button at the top of the page.

## Desired behavior

iOS: When a screen reader user is navigating each line item in the Delayed Refill message, the screen reader should only focus on the name/link of the prescription, not the bullet point.

Android: When a screen reader user taps on a link that directs the user to another page, the first item that the screen reader should focus on is the 'Back' button at the top of the page. This is consistent behavior with how the user navigates between other pages in the app.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High **Low** 
- Frequency: High **Low** 

- 3 - **Low**
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

iOS:
https://github.com/user-attachments/assets/a1316c6f-8771-4582-afbc-248d68b3d92b

Android:
https://github.com/user-attachments/assets/32f20295-a732-44ff-baf9-353c478cda35

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, global, Health, relevant feature, accessibility, etc)
- [ ] Estimate of 1 added (for front-end tickets)
- [ ] Added either to the relevant feature epic (if found during new feature implementation), or the relevant team's bug epic ([Global](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8302), [Health & Benefits](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8303), [API](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8304), [QART](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8305))

**Comment by SKMOstudio** [comment]
Closed

The iOS issue has already been resolved.

Upon further investigation, the scope of the Android issue extends to the Android app in general, not just the Delayed Refill message. Please see the new ticket #13221 for details. @matthew-df
